signer/src/testing/transaction_coordinator.rs

```diff
@@ -159,11 +165,13 @@ where
             .once()
             .returning(|_| Box::pin(async { Ok(None) }));
 
-        let (broadcasted_tx_sender, mut broadcasted_tx_receiver) = tokio::sync::mpsc::channel(1);
+        // TODO: multiple transactions can be generated and keeping this
```

Comment:
Is this exclusive to test code or is there something bigger that needs to be looked into?

Reply:
I believe that it is exclusive to the test code. I'm pretty sure that these senders/receivers are only compiled with the testing feature.

Reply:
Just double checked, and this channel is local to this particular test, not elsewhere.